frontend/pages/index.vue

```diff
@@ -1,3 +1,132 @@
 <template>
-  <h1>Home Page</h1>
+  <v-container fluid>
```

Comment:
I think a non-fluid container may be better to get a more consistent UI (a specific set of screen widths instead of every screen width)